csharp/Platform.IO/ConsoleHelpers.cs

```diff
@@ -10,16 +10,55 @@ namespace Platform.IO
 {
     public static class ConsoleHelpers
     {
+        /// <summary>
+        /// <para>Displays in the console appealing to press any key for continue</para>
+        /// <para>Выводит в консоли призыв нажать любую клавишу для продолжения</para>
+        /// </summary>
+        /// <returns>
+        /// <para>Returns true if a user pressed any key.</para>
+        /// <para>Возвращает истину если пользователь нажал любую клавишу</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static void PressAnyKeyToContinue()
         {
             Console.WriteLine("Press any key to continue.");
             Console.ReadKey();
         }
 
+        /// <summary>
+        /// <para>Calls GetOrReadArgument function with readMessage argument's value equal to index+1 argument</para>
```

Comment:
Just copy comments from the the method below. No need to comment how methods call each other.